performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/ConfigurationResponse.java

```diff
@@ -3,32 +3,66 @@
 import android.os.Bundle;
 import android.os.Parcel;
 import android.os.Parcelable;
-import com.google.gson.annotations.SerializedName;
+import android.support.annotation.NonNull;
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
+import org.json.JSONException;
+import org.json.JSONObject;
 
-/** Configuration Result */
-class ConfigurationResult implements Parcelable {
+/**
+ * Configuration Result
+ */
+class ConfigurationResponse implements Parcelable {
 
-  @SerializedName("enablePercent")
   private double enablePercent;
-
-  @SerializedName("enableNonMetricMeasurement")
   private boolean enableNonMetricMeasurement;
-
-  @SerializedName("sendUrl")
   private String sendUrl;
-
-  @SerializedName("sendHeaders")
   private Map<String, String> header;
-
-  @SerializedName("modules")
   private Map<String, String> modules;
 
   private static final String ENABLE_PERFORMANCE_TRACKING_KEY = "enablePerformanceTracking";
   private static final String ENABLE_RAT_KEY = "enableRat";
 
-  private ConfigurationResult(Parcel in) {
+  ConfigurationResponse(@NonNull String json) {
+    try {
+      JSONObject obj = new JSONObject(json);
+      enablePercent = obj.getDouble("enablePercent");
+      enableNonMetricMeasurement = obj.getBoolean("enableNonMetricMeasurement");
+      sendUrl = obj.getString("sendUrl");
+      header = toStringMap(obj.getJSONObject("sendHeaders"));
+      modules = toStringMap(obj.getJSONObject("modules"));
+    } catch (JSONException ignored) {
```

Comment:
Why ignore this instead of logging a debug message?

Reply:
added logging.
the logging in the runtime module is really lackluster, i think we should add a logger class like analytics has and streamline the whole affair.